### PR TITLE
fix notifications

### PR DESCRIPTION
**Pull request by nickk-o**

### CODE REVIEW

File: src/css/utils/variables.css [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
   /* stars */
   --stars-color-activ: #eea10c;
-
+  color: #0ecc2e;
```
Comment (OstafiichukO) [inline src/css/utils/variables.css:19]: not a variable

File: src/js/modalRating.js [hunk 2]
```diff
@@ -79,54 +94,57 @@ document.addEventListener('DOMContentLoaded', function() {
...
+
         try {
           const response = await axios.patch(url, {
             rate: starRating,
             email: refs.emailInput.value,
             review: refs.commentInput.value,
           });
           if (response.status === 200) {
-            iziToast.success({ message: 'Rating submitted successfully!' });
+            showMessage('success', 'Rating submitted successfully!');
```
Comment (OstafiichukO) [inline src/js/modalRating.js:132]: is title required here?

File: src/js/modalRating.js [hunk 3]
```diff
@@ -79,54 +94,57 @@ document.addEventListener('DOMContentLoaded', function() {
...
           } else {
-            iziToast.error({ title: 'Error', message: 'Failed to submit rating' });
+            showMessage('error', 'Failed to submit rating');
```
Comment (OstafiichukO) [inline src/js/modalRating.js:134]: is title required here?

File: src/js/modalRating.js [hunk 4]
```diff
@@ -79,54 +94,57 @@ document.addEventListener('DOMContentLoaded', function() {
...
-        event.preventDefault()
+      const handleSubmit = async event => {
+        event.preventDefault();
         const starId = selectedStar?.id;
         const starRating = parseInt(starId?.replace('star', ''), 10);
-      
+
         if (!starRating || isNaN(starRating)) {
-          iziToast.show({ message: 'Please add your rating' });
+          showMessage('info', 'Please add your rating');
```
Comment (OstafiichukO) [inline src/js/modalRating.js:108]: is title required here?

**Review by OstafiichukO — CHANGES_REQUESTED** [review]

File: src/js/modalRating.js (hunk 4 above)
Comment (nickk-o) [reply src/js/modalRating.js:108]: title is used for function to determine type of message, but won't be shown

File: src/css/utils/variables.css (hunk 1 above)
Comment (nickk-o) [reply src/css/utils/variables.css:19]: fixed

**Review by OstafiichukO — APPROVED** [review]

File: src/js/exercises.js [hunk 5]
```diff
@@ -52,7 +52,8 @@ export async function fetchData(type, obj) {
       }
     })
     .catch(err => {
-      showMessage('error', err.message);
+      console.log(err.message);
```
Comment (OstafiichukO) [inline src/js/exercises.js:55]: @mykola-ovchynnik  do we need those console logs?

Comment (OstafiichukO) [reply src/js/exercises.js:55]: @KorbetMaryna

**Pull request merged**